tests/test_cli.py

```diff
@@ -197,9 +197,14 @@ def test_log_encoding(self, caplog, monkeypatch):
                 self.exec_cli(command)
 
         # first message should be 'log encoding: <encoding>'
-        match = caplog.records()[0].message.split(':')
+        match = None
+        for record in caplog.records():
```

Comment:
Agreed. I think we should drop the 'flexibility when testing for log encoding' commit.